src/lib/blueprint.test.ts

```diff
@@ -343,3 +345,49 @@ test('getPreferredMethod: delete and post', (t) => {
     'Preferred method should be POST when both DELETE and POST are available',
   )
 })
+
+test('getWorkspaceScope: no auth methods', (t) => {
+  const authMethods: OpenapiAuthMethod[] = []
+  t.is(
+    getWorkspaceScope(authMethods),
+    'none',
+    'Workspace scope should be "none" when no auth methods are present',
+  )
+})
+
+test('getWorkspaceScope: only unscoped auth methods', (t) => {
+  const authMethods: OpenapiAuthMethod[] = [
+    'pat_without_workspace',
+    'console_session_token_without_workspace',
+  ]
+  t.is(
+    getWorkspaceScope(authMethods),
+    'none',
+    'Workspace scope should be "none" when only unscoped auth methods are present',
+  )
+})
+
+test('getWorkspaceScope: only scoped auth methods', (t) => {
+  const authMethods: OpenapiAuthMethod[] = [
+    'api_key',
+    'client_session',
+    'pat_with_workspace',
+  ]
+  t.is(
+    getWorkspaceScope(authMethods),
+    'required',
+    'Workspace scope should be "required" when only scoped auth methods are present',
+  )
+})
+
+test('getWorkspaceScope: both scoped and unscoped auth methods', (t) => {
+  const authMethods: OpenapiAuthMethod[] = [
+    'pat_with_workspace',
+    'pat_without_workspace',
+  ]
+  t.is(
+    getWorkspaceScope(authMethods),
+    'optional',
+    'Workspace scope should be "optional" when both scoped and unscoped auth methods are present',
+  )
+})
```

Comment:
For unit tests like this can be moved to https://github.com/seamapi/blueprint/blob/main/src/lib/blueprint.test.ts

Reply:
That's the same file as this one unless I'm missing something.